gui/actionables/actionables.md

```diff
@@ -0,0 +1,108 @@
+# Actionables
+
+## Overview
+
+Actionables provide a way to create custom triggers and menus that key on any text rendered in a query and take one or more actions when selected. Similar to an HTML Hyperlink, actionables can be used to open external URLs that key on data, but actionables can also be leveraged to submit new Gravwell queries, launch dashboards, and execute templates.
+
+Actionables are created by specifying one or more regular expressions, along with one or more actions. Gravwell automatically parses all text rendered with the [table](#!/search/table/table.md) and [chart](#!/search/chart/chart.md) renderers and creates menus for any provided triggers.
```

Comment:
i don't believe a single actionable can have more than one regex

Reply:
it can.